docs/optlib.rst

```diff
@@ -615,7 +622,8 @@ Multiline pattern flags
 	start position of the whole regex pattern). You do not need to add it for
 	the multi-table ``--_mtable-regex-<LANG>``.
 
-.. Q: isn't the above restriction really a bug? I think it is. I should fix it.
+.. TODO: Q: isn't the above restriction really a bug? I think it is. I should fix it.
+   Q to @masatake-san: Do you mean that {mgroup=0} can be omitted?
```

Comment:
Thank you very much for pointing out this.
I found a critical bug in the code about mgroup.
I will fix.

Reply:
I must revise how the group numbers are used in lregex.c.
However, the changes resulted from revising may conflict with the changes for optscript.